src/content/plots/CornerPlot.tsx

```diff
@@ -15,7 +15,7 @@ type ConerPlotPropType = {
     parameters: string[]
 }
 
-const corner_plot_size = 800
+const corner_plot_size = 500
```

Comment:
Not sure how strict we should be or exact what style we are using, but constants are usually written in all capital letters.

Reply:
I agree, change made